src/discretization/discretize_vars.jl

```diff
@@ -83,6 +84,7 @@ struct DiscreteSpace{N,M,G}
     Iaxies
     Igrid
     x2i
+    errvar # The error variable for the discrete space
```

Comment:
What is this?

Reply:
I added this to track when schemes were applied outside of their range of validity. It has served its purpose, I will remove.